CONTRIBUTING.md

```diff
@@ -1,139 +1,79 @@
+         upstream  git@github.com:GoogleCloudPlatform/forseti-security.git (fetch)
+         upstream  git@github.com:GoogleCloudPlatform/forseti-security.git (push)
 
-    All submissions, including submissions by project members, require a code review.
-    To begin the review process, create a new GitHub pull request. The GitHub UI
-    will show if there are any merge conflict(s) to be resolved.
+  1. Fetch the latest changes from upstream into your cloned repository:
 
-    The GitHub Pull Request UI will show you dropdowns to select the destination of
-    your pull request:
-    * the base fork is the upstream
-    * the head fork is your fork with your changes
+         git fetch upstream
+         git checkout master
+         git merge upstream/master
 
-    All tests must pass before we will review your PR.
-    We also check the codebase with pylint. You can install the git hooks provided
-    under scripts/githooks/ to do the check locally.
+  1. Create a new local development branch for your feature or bug fix:
 
-9. Merging your PR.
+         git checkout -b YOUR_DEVELOPMENT_BRANCH
 
-    Once your PR is approved and all merge conflicts are resolved, we will merge your PR.
+  1. Create your change.
+    - Don't incorporate multiple changes in one PR. A change should be a logical, self-contained unit of work, feature, or fix. This simplifies troubleshooting and rollbacks.
+    - Learn how to execute [Inventory](inventory-quickstart), [Scanner](scanner-quickstart), or [Enforcer](enforcer-quickstart).
+  1. Create your test.
+    - Unit tests should provide high and useful coverage. If your change involves substantial logic, we'll ask you to write applicable unit tests.
+    - Unit tests should be written with google-apputils basetest framework. For an example of how to use the framework, see [Google-Style Tests](https://pypi.python.org/pypi/google-apputils).
+    - Learn how to [run the tests](testing).
+  1. Commit your changes and push them to your development branch:
 
+         git push origin YOUR_DEVELOPMENT_BRANCH
 
-[GitHub Help]: https://help.github.com/articles/about-pull-requests/
-[Google Python Style Guide]: https://google.github.io/styleguide/pyguide.html
-[Inventory]: https://googlecloudplatform.github.io/forseti-security/modules/core/inventory/
-[Scanner]: https://googlecloudplatform.github.io/forseti-security/modules/core/scanner/
-[Enforcer]: https://googlecloudplatform.github.io/forseti-security/modules/core/enforcer/
-[basic example]: https://pypi.python.org/pypi/google-apputils
-[Instructions to run the tests.]: https://googlecloudplatform.github.io/forseti-security/testing/
-[forseti-security@google.com]: mailto:forseti-security@google.com
+  1. Open a [pull request](https://help.github.com/articles/about-pull-requests/) to begin the code review.
+    - All submissions, including submissions by project members, require a code review.
+    - The GitHub UI will display any merge conflicts you need to resolve.
```

Comment:
The Github UI will display whether you have any merge conflicts that need to be resolved.